lib/launch-local.js

```diff
@@ -172,7 +173,7 @@ class Launcher {
       return {};
     }
 
-    const runnableScript = prepareScript(this.script, this.payload, this.opts);
+    const runnableScript = prepareScript(this.script, _.cloneDeep(this.payload), this.opts);
```

Comment:
any reason not to clone `payload` inside `prepareScript`?

Reply:
`prepareScript` is called by each worker, and by the time we are there the payload is already a copy so cloning it seemed redundant. it only needs to be cloned when the main thread uses it